R/Rsw.R

```diff
@@ -311,6 +313,11 @@ sw_exec <- function(
 
   # Run SOILWAT2
   res <- .Call(C_start, input, inputData, weatherList, quiet)
+
+  if (is.null(res)) {
```

Comment:
Can we test that the error/capture/cleanup approach introduced with commit 362fada5ed00b72178bd61f29215bbecbe0e7020 actually works?

I'm not so sure anymore that this was a good idea. I think we need to explore what `.Call()` is returning after SOILWAT2 called `error()` in `LogError()`. 

Also see https://cran.r-project.org/doc/manuals/R-exts.html#Condition-handling-and-cleanup-code